__tests__/app.test.js

```diff
@@ -163,7 +163,24 @@ describe("GET/api/articles/:article_id", () => {
         });
       });
   });
+  test('Filters articles by topics query', () => {
+    return request(app)
+    .get('/api/articles?topic=cats')
+    .expect(200)
+    .then((response) => {
+      const articles = response.body.articles;
+      expect(articles).toHaveLength(1)
```

Comment:
Its important here to check that any returned articles have the topic you expect